evil-commands.el

```diff
@@ -3398,6 +3398,16 @@ If FORCE is non-nil and MARKS is blank, all local marks except 0-9 are removed."
             (forward-line (1- line))))
       (user-error "File does not exist."))))
 
+(evil-define-command evil-find-file-at-point-visual (&optional create-on-error)
```

Comment:
What's `create-on-error` emulating of vim? `:e <cfile>`? As it's not a feature of gf in vim, and I can't see you having used it anywhere (yagni), I'm not sure it should be part of the PR.?